pkg/aggregation/table.go

```diff
@@ -67,7 +70,15 @@ func (s *TableAggregator) SampleItem(colKey, rowKey string, inc int64) {
 		s.rows[rowKey] = row
 	}
 
-	row.cols[colKey] += inc
+	curr := row.cols[colKey]
+	curr += inc
+	if curr > s.max {
```

Comment:
Consider benchmarking this. It's only effectively used for display, so might be better to compute on-demand (and more accurate) than tracking it